website/blog/2022-04-11-h2-qc-updated-sumstats.md

```diff
@@ -0,0 +1,74 @@
+
+We have made our final heritability estimates available for download in the [phenotype manifest](ADDMANIFEST) and data from all methods available in the [heritability manifest](ADDMANIFEST). All results are also available in [MatrixTable format](https://pan.ukbb.broadinstitute.org/downloads). Further description of the heritability estimation approach can be found [here](https://pan.ukbb.broadinstitute.org/docs/heritability).
+
+## QC approach
+
+We observed significantly negative RHEmc heritability estimates for traits that were likely to have a high propensity of confounding (e.g., UK geographic location). We subsequently developed a systematic QC approach using the following *sequential* filters to prioritize high quality phenotypes for further analysis:
+
+1. `GWAS_run`: if the GWAS was performed for the ancestry-trait pair
+2. `ancestry_reasonable_n`: if the ancestry has a reasonable sample size; has the effect of removing AMR
+3. `defined_h2`: if the heritability estimate is non-missing
+4. `significant_z`: if the ancestry-trait pair shows $h^2$ z-score $> 0$
+5. `in_bounds_h2`: if, for all ancestries for a given trait, observed-scale heritability estimates $\in (0,1)$
+6. `normal_lambda`: if, for all ancestries for a given trait, $\lambda_{GC} > 0.9$
+7. `normal_ratio`: if, for the top three best powered ancestry groups (EUR, CSA, AFR), the S-LDSC ratio, given by $\frac{intercept-1}{mean \chi^2 -1}$, $< 0.3$ or the ratio z-score $< 4$
+8. `EUR_plus_1`: if the trait passes all above filters in EUR and at least 1 other ancestry group
+
+A more detailed description of the approach is forthcoming. All QC results for ancestry-trait pairs can be found in the [phenotype manifest](ADDMANIFEST) and [heritability manifest](ADDMANIFEST).
+
+## Maximally independent set
+
+Using the set of phenotypes with ancestries passing all of the above QC filters, we next wanted to construct a set of independent phenotypes to avoid double-counting in downstream analyses. To this end we constructed a maximally independent set of phenotypes using a pairwise phenotypic correlation matrix (released [here](https://pan.ukbb.broadinstitute.org/downloads) via [`make_pairwise_ht`](https://github.com/Nealelab/ukb_common/blob/f9b4c037b57e932a52dcfb8c35f1e077c6939610/src/ukbb_common/utils/phenotype_loading.py#L338)). Of all phenotype pairs, we retained any with a pairwise correlation $r < 0.1$. For pairs with  $r > 0.1$ , we used [`hl.maximal_independent_set`](https://hail.is/docs/0.2/methods/misc.html#hail.methods.maximal_independent_set) to identify indendent phenotypes for retention, imposing a tiebreaker of higher case count (or higher sample size for continuous phenotypes). This resulted in the selection of 195 independent phenotypes, which were used in subsequent analyses analyzing trends across phenotypes. We have released this set in the [phenotype manifest](ADDMANIFEST).
```

Comment:
how many pop-pheno pairs?

Reply:
I don't think pairs make sense here because the maximally independent set process operated on whole phenotypes; maybe @eatkinson can comment?

Reply:
Yes the phenotype pairwise correlations are computed across everyone